locale/de.properties

```diff
@@ -0,0 +1,69 @@
+activate_on_site = Klicke hier um den Privacy Badger auf dieser Seite zu aktiveren.
+click_badger_activate_on_site = Klicke auf das Dachs-Symbol um den Privacy Badger auf dieser Seite zu aktivieren.
+cookie_prefs_warning = <p id="cookiePrefsWarning">
+Achtung: Der Privacy Badger Alpha arbeitet vielleicht nicht wie erwartet mit
```

Comment:
At first I would not add an article before "Privacy Badger". Privacy Badger is a proper name.
I also don't know why it is still called "alpha" there.